tests/e2e/specs/stories-editor/resizing.js

```diff
@@ -269,34 +269,78 @@ describe( 'Resizing', () => {
+		} );
 
+		it( 'should not resize smaller than the set minimum width: left', async () => {
 			const resizableHandleLeft = await page.$( '.wp-block.is-selected .components-resizable-box__handle-left' );
 			await dragAndResize( resizableHandleLeft, { x: 300, y: 0 } );
-			dimensions = await getSelectedBlockDimensions();
+			const dimensions = await getSelectedBlockDimensions();
 			expect( dimensions.width ).toStrictEqual( textBlockMinWidth );
+		} );
 
+		it( 'should not resize smaller than the set minimum width: right', async () => {
 			const resizableHandleRight = await page.$( '.wp-block.is-selected .components-resizable-box__handle-right' );
 			await dragAndResize( resizableHandleRight, { x: -300, y: 0 } );
-			dimensions = await getSelectedBlockDimensions();
+			const dimensions = await getSelectedBlockDimensions();
 			expect( dimensions.width ).toStrictEqual( textBlockMinWidth );
 		} );
+
+		it( 'should not move the position of the block when the width is not changing: left', async () => {
+			// Ensure first that the block is already minimum width.
+			const resizableHandleLeft = await page.$( '.wp-block.is-selected .components-resizable-box__handle-left' );
+			await dragAndResize( resizableHandleLeft, { x: 300, y: 0 } );
+			const dimensions = await getSelectedBlockDimensions();
+			expect( dimensions.width ).toStrictEqual( textBlockMinWidth );
+
+			// Get the initial position.
+			const { positionLeft: positionLeftBefore } = await getSelectedBlockPosition();
+			// Try resizing again.
+			await dragAndResize( resizableHandleLeft, { x: 300, y: 0 } );
+			const { positionLeft } = await getSelectedBlockPosition();
+			expect( positionLeft ).toStrictEqual( positionLeftBefore );
+
+			// Rotate the block and try again.
+			await rotateSelectedBlock();
+			await dragAndResize( resizableHandleLeft, { x: 300, y: 0 } );
+			const { positionLeft: positionLeftRotated } = await getSelectedBlockPosition();
+			expect( positionLeftRotated ).toStrictEqual( positionLeftBefore );
+		} );
+
+		it( 'should not move the position of the block when the height is not changing: top', async () => {
+			// Ensure first that the block is already minimum height.
+			const resizableHandleTop = await page.$( '.wp-block.is-selected .components-resizable-box__handle-top' );
+			await dragAndResize( resizableHandleTop, { x: 0, y: 250 } );
+			const dimensions = await getSelectedBlockDimensions();
+			expect( dimensions.height ).toStrictEqual( textBlockMinHeight );
+
+			// Get the initial position.
+			const { positionTop: positionTopBefore } = await getSelectedBlockPosition();
+			// Try resizing again.
+			await dragAndResize( resizableHandleTop, { x: 0, y: 250 } );
+			const { positionTop } = await getSelectedBlockPosition();
+			expect( positionTop ).toStrictEqual( positionTopBefore );
+
+			// Rotate the block and try again.
+			await rotateSelectedBlock();
+			await dragAndResize( resizableHandleTop, { x: 300, y: 0 } );
```

Comment:
Here you switch to dragging in `x` direction in stead of `y` direction after rotation. But in the above test, you drag in `x` both before and after rotation. I don't know which is the right one, but it seems to be this one, perhaps?

I guess that depends on how the item is rotated of course 😝.

Reply:
Hah! Good catch, thank you. The rotation is not enough for the initial direction not to work -- will leave a comment with degrees.